src/com/severalcircles/colors/events/ChatEvent.java

```diff
@@ -10,7 +10,7 @@
 public class ChatEvent implements Listener {
     @EventHandler
     public void onPlayerChat(AsyncPlayerChatEvent event) {
-
+        System.out.println("Chat event triggered");
```

Comment:
I am wondering how this got into this pull but you know

Reply:
What is that? I don't remember adding that.